openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -432,15 +432,16 @@ const EntityTable = ({
             lowerCase(dataTypeDisplay)
           ) : (
             <PopOver
+              delay={200}
               html={
                 <div className="break-word">
                   <span>{lowerCase(dataTypeDisplay)}</span>
                 </div>
               }
               key="pop-over"
-              position="bottom"
+              position="top"
               theme="light"
-              trigger="click">
+              trigger="mouseenter">
```

Comment:
Please use AntD `Popover`

Reply:
@chirag-madlani Updated to AntD `Popover`